lib/mailbox-handler.js

```diff
@@ -26,7 +26,7 @@ class MailboxHandler {
 
     create(user, path, opts, callback) {
         this.createAsync(user, path, opts)
-            .then(mailboxData => callback(null, ...Object.values(mailboxData)))
+            .then(mailboxData => callback(null, ...[mailboxData.status, mailboxData.id]))
```

Comment:
I see that you prefer to use the spread operator, even if you don't really need it 😄

Reply:
Ah yes haha. Indeed here it is not required, I could just pass the parameters as they are. Will fix in a sec

Reply:
No need, already merged